MPF/ViewModels/MainViewModel.cs

```diff
@@ -570,6 +570,53 @@ public async void InitializeUIValues(bool removeEventHandlers, bool rescanDrives
             App.Instance.StartStopButton.IsEnabled = ShouldEnableDumpingButton();
         }
 
+        /// <summary>
+        /// Performs a fast update of the output path while skipping disc checks
+        /// </summary>
+        /// <param name="removeEventHandlers">Whether event handlers need to be removed first</param>
+        private void FastUpdateLabel(bool removeEventHandlers)
+        {
+            // Disable the dumping button
+            App.Instance.StartStopButton.IsEnabled = false;
+
+            // Safely uncheck the parameters box, just in case
+            if (App.Instance.EnableParametersCheckBox.IsChecked == true)
+            {
+                App.Instance.EnableParametersCheckBox.Checked -= EnableParametersCheckBoxClick;
+                App.Instance.EnableParametersCheckBox.IsChecked = false;
+                App.Instance.ParametersTextBox.IsEnabled = false;
+                App.Instance.EnableParametersCheckBox.Checked += EnableParametersCheckBoxClick;
+            }
+
+            // Set the UI color scheme according to the options
+            if (App.Options.EnableDarkMode)
+                EnableDarkMode();
+            else
+                DisableDarkMode();
+
+            // Force the UI to reload after applying the theme
+            App.Instance.UpdateLayout();
```

Comment:
This part is only needed if the options updated. You should be in the clear to remove this.